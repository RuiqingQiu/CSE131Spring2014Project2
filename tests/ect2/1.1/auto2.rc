function : int main(){
   const auto f = -801.3;
   const auto i = -0xFAFA;
   const auto b = true;
   cout << b << f << i << endl;
   return 0;
}
