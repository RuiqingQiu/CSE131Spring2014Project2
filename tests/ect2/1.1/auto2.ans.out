true-801.30-64250
